ionic2/app/components/poke-poi-card/poke-poi-card.component.ts

```diff
@@ -0,0 +1,91 @@
+import { Component, Input, ViewChild, OnInit, ElementRef, ChangeDetectorRef,
+  animate, trigger, state, style, transition } from '@angular/core';
+import { Events, NavController } from 'ionic-angular';
+import { Subscription } from 'rxjs';
+
+import { ApiService } from '../../services/api.service';
+import { PokePOIBubbleComponent } from '../poke-poi-bubble/poke-poi-bubble.component';
+import { PokeDetailPage } from '../../pages/poke-detail/poke-detail.page';
+import { PokeTypeComponent } from '../poke-details/poke-type.component';
+import { PokeSighting } from '../../models/poke-sighting';
+
+let Hammer = require('hammerjs');
+
+@Component({
+  template: require('./poke-poi-card.component.html'),
+  styles: [require('./poke-poi-card.component.scss')],
+  selector: 'poke-poi-card',
+  directives: [
+    PokePOIBubbleComponent,
+    PokeTypeComponent
+  ],
+  animations: [
+    trigger('slide', [
+      state('visible', style({transform: 'translateY(0)'})),
+      state('hidden', style({transform: 'translateY(120%)'})),
+      transition('visible <=> hidden', animate('300ms ease')),
+    ])
+  ]
+})
+export class PokePOICardComponent implements OnInit {
+
+  @ViewChild('slideCard') slideCard: ElementRef;
+
+  pokePOI: PokeSighting;
+  loadPokemon: Subscription;
+  slideState: string = 'hidden';
+
+  constructor(private navCtrl: NavController,
+              private apiService: ApiService,
+              private changeDetectorRef: ChangeDetectorRef) {}
+
+  ngOnInit(): any {
```

Comment:
For consistency reasons, I'd suggest to not annotate Lifecycle methods, you haven't done it in the other components and there's no real benefit for annotating any either, esp if it's always void.